middleware/retry/exponential_backoff_test.go

```diff
@@ -0,0 +1,66 @@
+package retry
+
+import (
+	"fmt"
+	"github.com/stretchr/testify/assert"
```

Comment:
I think you should use fiber's assert instead of adding new deps.

Reply:
Fixed in cd5d0cd.

Reply:
Dont forget running go mod tidy